ui/src/app/job-details/resources/resources-table/resources-table.component.html

```diff
@@ -0,0 +1,13 @@
+<mat-table class="resources-table" #table [dataSource]="entryKeys">
+  <ng-container matColumnDef="key">
+    <mat-cell *matCellDef="let key"> {{key}} </mat-cell>
```

Comment:
nit: Please add spaces around the `key` variable for consistency

Reply:
Done.